README.md

```diff
@@ -27,16 +27,14 @@ _From [Mux](https://mux.com?utm_source=github&utm_medium=social&utm_campaign=med
 
 Visit the [official documentation for Media Chrome](https://media-chrome.org/en/get-started) for the latest up-to-date usage instructions.
 
-You can find the [current release candidate documentation here.](https://v1.media-chrome.org/en/get-started)
-
 ## Examples
 
 <a href="https://media-chrome.mux.dev/examples/vanilla/" target="_blank">See all of the repo examples in action.</a>
 
 ### Video Example
 
 ```html
-<script type="module" src="https://cdn.jsdelivr.net/npm/media-chrome@0.21/+esm"></script>
+<script type="module" src="https://cdn.jsdelivr.net/npm/media-chrome@1.0/+esm"></script>
```

Comment:
do we want these pointing at the major or the minor? Less risk if we pin to a minor, but then we have to be more vigilant in keeping the links up to date.